src/Handlers/ResultHandler.php

```diff
@@ -95,71 +96,57 @@ public function __construct(
      * @param AsyncPaymentTransactionStruct $transaction
      * @param Request $request
      * @param SalesChannelContext $salesChannelContext
-     * @throws PaymentCancelledException
-     * @throws PaymentFailedException
+     * @throws PaymentException
      */
     public function processResult(
         AsyncPaymentTransactionStruct $transaction,
         Request $request,
         SalesChannelContext $salesChannelContext
     ) {
-        // Retrieve paymentResponse and if it is
-        $paymentResponse = $this->paymentResponseService->getWithOrderTransaction(
-            $transaction->getOrderTransaction()
-        );
+        // Retrieve paymentResponse and if it exists
+        $paymentResponse = $this->paymentResponseService->getWithOrderTransaction($transaction->getOrderTransaction());
 
-        $result = $this->paymentResponseHandlerResult->createFromPaymentResponse(
-            $paymentResponse
-        );
-
-        if ('RedirectShopper' === $result->getResultCode()) {
-            // Validate 3DS1 parameters
-            // Get MD and PaRes to be validated
```

Comment:
`details.payload`, `details.MD` and `details.PaRes` no longer need to be sent to the `/payment/details` endpoint in v67. The only parameter we need to send here is `details.redirectResult` (`details.threeDSResult` is sent by the checkout component after 3ds2 user confirmation)
https://docs.adyen.com/online-payments/release-notes?tab=script_1666a37f_2#checkout-api-v67